roles/openshift_ca/tasks/main.yml

```diff
@@ -85,6 +98,19 @@
     use_regex: true
   register: g_master_legacy_ca_result
 
+# This should NOT replace the CA due to --overwrite=false when a CA already exists.
+- name: Create the front-proxy CA if it does not already exist
+  command: >
+    {{ hostvars[openshift_ca_host]['first_master_client_binary'] }} adm ca create-signer-cert
+    --cert="{{ openshift_ca_config_dir }}/front-proxy-ca.crt"
+    --key="{{ openshift_ca_config_dir }}/front-proxy-ca.key"
+    --serial="{{ openshift_ca_config_dir }}/ca.serial.txt"
+    --expire-days={{ openshift_ca_cert_expire_days }}
+    --overwrite=false
```

Comment:
does this interfere with redeploying certs? wouldn't we want to overwrite if we are forcing a cert redeploy?

Reply:
Same logic as for master cert redeploy - I *think* a higher level playbook wipes the directory.